src/character.cpp

```diff
@@ -11461,8 +11462,8 @@ int Character::run_cost( int base_cost, bool diag ) const
     const bool flatground = movecost < 105;
     map &here = get_map();
     // The "FLAT" tag includes soft surfaces, so not a good fit.
-    const bool on_road = flatground && here.has_flag( "ROAD", pos() );
-    const bool on_fungus = here.has_flag_ter_or_furn( "FUNGUS", pos() );
+    const bool on_road = flatground && here.has_flag( STATIC( "ROAD" ), pos() );
+    const bool on_fungus = here.has_flag_ter_or_furn( STATIC( "FUNGUS" ), pos() );
```

Comment:
These changes don't make sense to me as given.  Doesn't it have to be `STATIC( std::string( ... ) )` to have a performance benefit?

Reply:
Yes, you're correct. 

 I believe the comment to the `STATIC` macro should be changed to reflect that.

Reply:
I'll do a follow up PR today.